README.md

```diff
@@ -186,6 +186,43 @@ goworker.Enqueue(&goworker.Job{
 })
 ```
 
+## Redis Settings
+
+Rather than specify a full URI, you may use the `RedisSettings` struct to specify more granular optios for the redis connection.
+
+```go
+func init() {
+	redisSettings := goworker.RedisSettings {
+		DB:       "2",
+		Host:     "localhost:6379",
+		Scheme:   "redis",
+		Password: "Foobar"
+	}
+	settings := goworker.WorkerSettings{
+		Connections:    100,
+		Queues:         []string{"myqueue", "delimited", "queues"},
+		UseNumber:      true,
+		ExitOnComplete: false,
+		Concurrency:    2,
+		Namespace:      "resque:",
+		RedisSettings:  redisSettings,
+	}
+	goworker.SetSettings(settings)
+	goworker.Register("MyClass", myFunc)
+}
+```
+
+Goworker uses these options to support [Redis-Sentinel](https://redis.io/topics/sentinel). Here is an example of redisSettings that work with Redis-Sentinel:
```

Comment:
Two minor points:
* It's "Redis Sentinel", not "Redis-Sentinel";  two places in this sentence should be corrected.
* You probably want "RedisSettings" rather than "redisSettings"; the former is the type of object we're talking about, the latter is just an example-specific variable name.